docs/user/main.py

```diff
@@ -1,15 +1,6 @@
 from pathlib import Path
 
-INDEX_ATTESTATIONS = """
-!!! info
-
-    Index attestations are currently under active development,
-    and are not yet considered stable.
-"""
-
-PREVIEW_FEATURES = {
-    "index-attestations": INDEX_ATTESTATIONS,
-}
+PREVIEW_FEATURES = {}
```

Comment:
Note: this is now unused, but I figured it's better to keep in as scaffolding for future preview features.